src/gsi/get_gefs_for_regional.f90

```diff
@@ -336,12 +353,82 @@ subroutine get_gefs_for_regional
      if(mype == 0) then
         write(6,*) ' nemsio: fhour,idate=',fhour,idate
         write(6,*) ' iadate(y,m,d,hr,min)=',iadate
-        write(6,*) ' nemsio: jcap,levs=',njcap,levs
+        write(6,*) ' nemsio: jcap,levs=',levs
         write(6,*) ' nemsio: latb,lonb=',latb,lonb
         write(6,*) ' nemsio: idvc,nvcoord=',idvc,nvcoord
         write(6,*) ' nemsio: idsl=',idsl
      end if
 
+!  add netCDF header information: 
+     else ! use_gfs_ncio and get this information
+        write(sfilename,'("sfcf",i2.2)')nhr_assimilation
+        ! open the netCDF file
+        atmges = open_dataset(filename)
+        !sfcges = open_dataset(sfilename)
```

Comment:
do you not need the sfcges file? if not, remove this line

Reply:
Yes,  I don't need the sfcges files now, removed.